src/components/SearchResults.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import "../styles/search-results.css";
+
+const SearchResults = ({ results }) => {
+  if (!results.length) {
+    return <p>No results found!</p>;
+  } else {
+    return (
+      <div className="image-container" data-testid="image-container">
+        {results.map((image, index) => (
+          <img
+            key={index}
+            className="space-image"
+            src={image}
+            alt="spaceImage"
```

Comment:
This alt tag isnt all that helpful. Their purpose is to describe what an image is for people using a screen reader, in this case all the images will have this generic label that doesnt help describe anything really. A good next step for future development would be to see if you can get the alt tag for each image from the API and attach it to its image